dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -49,6 +49,10 @@ write_default_zebra_config()
         echo "no fpm use-next-hop-groups" >> $FILE_NAME
         echo "fpm address 127.0.0.1" >> $FILE_NAME
     }
+
+    grep -q '^no zebra nexthop kernel enable' $FILE_NAME || {
```

Comment:
This logic will create problem when zebra_nexthop is enabled

Reply:
Thanks for reminder @dgsudharsan , this script should only be called during initialization. and the enable->disable case should not occur in production. Please correct me if anything wrong, thanks

Reply:
@dgsudharsan Could you please help to review the PR again when free, thank you very much